tools/crm_mon.c

```diff
@@ -804,21 +941,19 @@ main(int argc, char **argv)
 
     /* Extra sanity checks when in CGI mode */
     if (output_format == mon_output_cgi) {
-        argerr += (optind < argc);
-        argerr += (output_filename != NULL);
-        argerr += ((cib) && (cib->variant == cib_file));
-        argerr += (external_agent != NULL);
-        argerr += (daemonize == TRUE);  /* paranoia */
-
-    } else if (optind < argc) {
-        printf("non-option ARGV-elements: ");
-        while (optind < argc)
-            printf("%s ", argv[optind++]);
-        printf("\n");
-    }
-
-    if (argerr) {
-        return clean_up(CRM_EX_USAGE);
+        if (output_filename != NULL) {
+            fprintf(stderr, "CGI mode does not have an output filename set\n");
```

Comment:
cannot be used with

(though this will become irrelevant once we're using the new output model)